hnn_core/gui/gui.py

```diff
@@ -491,19 +610,23 @@ def compose(self, return_layout=True):
                 self._backend_config_out]),
         ], layout=self.layout['config_box'])
 
+        connectivity_configuration = Tab()
+
         connectivity_box = VBox([
             HBox([self.load_connectivity_button, ]),
             self._connectivity_out,
         ])
 
-        # accordions to group local-connectivity by cell type
-        connectivity_boxes = [
-            VBox(slider) for slider in self.connectivity_widgets]
-        connectivity_names = (
-            'Layer 2/3 Pyramidal', 'Layer 5 Pyramidal', 'Layer 2 Basket',
-            'Layer 5 Basket')
-        cell_connectivity = Accordion(children=connectivity_boxes)
```

Comment:
Was this somehow redundant or being unused? I see that you're using `add_connectivity_tab` to setup the connectivity, but that function already existed before this PR

Reply:
I dont think it was being used at all. That variable only exists during the scope of that function.